### PR TITLE
ARROW-16244: [Website] Arrow for R cheatsheet blog post

### PR DESCRIPTION
**Pull request by stephhazlitt**
Offers a blog post introducing the Arrow for R cheatsheet.

### CODE REVIEW

File: _posts/2022-04-19-arrow-r-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The Arrow for R cheatsheet is intended to be an easy-to-scan introduction to the Arrow R package and Arrow data structures, with getting started sections on some of the package’s main functionality. The cheatsheet includes introductory snippets on using Arrow to read and work with larger-than-memory multi-file data sets, sending and receiving data with Flight, reading data from cloud storage without downloading the data first, and more. The Arrow for R cheatsheet also directs users to the full [Arrow for R package documentation and articles](https://arrow.apache.org/docs/r/index.html) and the [Arrow Cookbook](https://arrow.apache.org/cookbook/), both full of code recipes to support users build their Arrow-based data workflows. Finally, the cheatsheet debuts one of the first uses of the hot-off-the-presses Arrow hex sticker, recently made available as part of the [Apache Arrow visual identity guidance](https://arrow.apache.org/visual_identity/).
```
Comment (thisisnic) [inline _posts/2022-04-19-arrow-r-cheatsheet.md:41]:
```suggestion
The Arrow for R cheatsheet is intended to be an easy-to-scan introduction to the Arrow R package and Arrow data structures, with getting started sections on some of the package’s main functionality. The cheatsheet includes introductory snippets on using Arrow to read and work with larger-than-memory multi-file data sets, sending and receiving data with Flight, reading data from cloud storage without downloading the data first, and more. The Arrow for R cheatsheet also directs users to the full [Arrow for R package documentation and articles](https://arrow.apache.org/docs/r/index.html) and the [Arrow Cookbook](https://arrow.apache.org/cookbook/r/), both full of code recipes to support users build their Arrow-based data workflows. Finally, the cheatsheet debuts one of the first uses of the hot-off-the-presses Arrow hex sticker, recently made available as part of the [Apache Arrow visual identity guidance](https://arrow.apache.org/visual_identity/).
```

Perhaps consider changing "both full of code recipes" to "both full of code examples and recipes" as the pedant in me says that it's only the cookbook which contains recipes.

**Review by thisisnic — COMMENTED** [review]
Love this!  Just a couple of minor suggestions.  I think the tone of this is wonderful!

File: _posts/2022-04-19-arrow-r-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Arrow for R Cheatsheet
+date: "2022-04-19 00:00:00"
```
Comment (jonkeane) [inline _posts/2022-04-19-arrow-r-cheatsheet.md:4]: We will want to update this + the filename right before we merge so that the publish date is right

**Review by jonkeane — APPROVED** [review]
This is fantastic, thanks for this! I'm happy to merge this, though we should bump the dates before we do.

File: _posts/2022-04-19-arrow-r-cheatsheet.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Arrow for R Cheatsheet
```
Comment (nealrichardson) [inline _posts/2022-04-19-arrow-r-cheatsheet.md:3]:
```suggestion
title: Apache Arrow for R Cheatsheet
```

File: _posts/2022-04-19-arrow-r-cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We are excited to introduce the new [Arrow for R Cheatsheet](https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow-cheatsheet.pdf).
```
Comment (nealrichardson) [inline _posts/2022-04-19-arrow-r-cheatsheet.md:27]:
```suggestion
We are excited to introduce the new [Apache Arrow for R Cheatsheet](https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow-cheatsheet.pdf).
```

**Review by nealrichardson — COMMENTED** [review]
Pedantic "Apache Arrow" suggestions for copyright compliance aside, this looks great, thanks!

File: _posts/2022-04-19-arrow-r-cheatsheet.md (hunk 2 above)
Comment (stephhazlitt) [reply _posts/2022-04-19-arrow-r-cheatsheet.md]: Updated to April 27, with the idea that we may merge on that date.

**Pull request merged**